src/core/css/inc/mixins.scss

```diff
@@ -77,8 +77,7 @@
       @content;
     }
   } @else {
-    @warn 'Unfortunately, no value could be retrieved from `#{$breakpoint}`. '
-      + 'Please make sure it is defined in `$breakpoints` map.';
+    @warn 'Breakpoint `#{$breakpoint}` not found in `$breakpoints` map.';
```

Comment:
Somewhat [surprisingly](https://github.com/mozilla/addons-frontend/commit/b4fbc2641061382b5d3fea8597bf52ad4e738e99) it wasn't a Canadian. I would have believed you though ;)